packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -40,7 +40,12 @@ test.serial(
 
     const materials = buildProposal(
       '@agoric/builders/scripts/fast-usdc/init-fast-usdc.js',
-      ['--oracle', 'a:agoric1watcher1'],
+      [
+        '--oracle',
+        'a:agoric1watcher1',
+        '--feedPolicy',
+        '{"nobleAgoricChannelId":"channel-21","nobleDomainId":4,"chainPolicies":{"Arbitrum":{"cctpTokenMessengerAddress":"0x19330d10D9Cc8751218eaf51E8885D058642E08A","chainId":42161,"confirmations":1,"nobleContractAddress":"0x19330d10D9Cc8751218eaf51E8885D058642E08A"}}}',
```

Comment:
This might read nicer using...

```suggestion
        JSON.stringify({"nobleAgoricChannelId":"channel-21","nobleDomainId":4,"chainPolicies":{"Arbitrum":{"cctpTokenMessengerAddress":"0x19330d10D9Cc8751218eaf51E8885D058642E08A","chainId":42161,"confirmations":1,"nobleContractAddress":"0x19330d10D9Cc8751218eaf51E8885D058642E08A"}}}),
```

and then let the linter pretty-print it.

Reply:
Done